applications/dashboard/controllers/api/MediaApiController.php

```diff
@@ -237,6 +237,9 @@ public function patch_attachment(int $id, array $body): array {
     public function post(array $body) {
         $this->permission('Garden.Uploads.Add');
 
+        //Bypass ImageUpload.Limits.Enabled if user has 'Garden.Community.Manage' permission or higher
```

Comment:
Move this call closer to where you are using it. Here is a quote:

> Keep Variables "Live" for as Short a Time as Possible
> A concept that's related to variable span is variable "live time," the total number of statements over which a variable is live. A variable's life begins at the first statement in which it's referenced; its life ends at the last statement in which it's referenced.

*McConnell, Steve. Code Complete (2nd Edition) (Developer Best Practices) (Kindle Locations 6160-6164). Pearson Education. Kindle Edition.*